src/casm/clex/ConfigIOStrucScore.cc

```diff
@@ -117,6 +120,11 @@ bool StrucScore::init(const Configuration &_tmplt) const {
 
   m_strucmapper = notstd::make_unique<StrucMapper>(
       PrimStrucMapCalculator(*m_altprim), m_strain_weight);
+
+  for (Index i=0; i < m_prop_names.size(); i++) {
```

Comment:
Looking at this again, overriding `VectorXdDatumFormatter::init()` prevents the automatic initialization of the index rules. Really, `VectorXdDatumFormatter::init()` should probably be marked `final` and call a separate virtual method (e.g., `VectorXdDatumFormatter::init_and_return_index_rules()`), so that implementers would be forced to consider the index rules.

Given that, I think this makes sense as is.